python/pyspark/util.py

```diff
@@ -503,11 +534,236 @@ def start(self) -> None:
         return super(InheritableThread, self).start()
 
 
+class PythonEvalType:
```

Comment:
Is this a breaking change? I.e the old import path will now throw an exception

Reply:
This isn't an API. `PandasUDFType` is the API.